packages/server/src/app.module.ts

```diff
@@ -30,8 +32,9 @@ import configuration from './config/configuration';
       inject: [ConfigService]
     }),
     OrganizationModule,
-    JupyterhubModule,
-    AuthModule
```

Comment:
Why is the auth module removed?

Reply:
`AuthModule` at line 13 and `AuthModule` at line 34 are different auth module?

Reply:
Opps, my bad, didnt see the duplications